packages/workbox-core/models/messages/messageGenerator.mjs

```diff
@@ -12,17 +12,14 @@ const fallback = (code, ...args) => {
 const generatorFunction = (code, ...args) => {
   const message = messages[code];
   if (!message) {
-    logger.warn(`Unable to find message for code '${code}'.`, args);
-    return fallback(code, ...args);
+    throw new Error(`Unable to find message for code '${code}'.`);
```

Comment:
Did you intend to remove the logging here?

Reply:
yes. This now throws to ensure we aren't silently failing to show a message when we intended to.